Resources/Prototypes/Body/Organs/vox.yml

```diff
@@ -1,3 +1,32 @@
+- type: entity # starlight start
+  parent: OrganHumanBrain
```

Comment:
Right now this is getting the tag "Meat" from OrganHumanBrain, which may lead to unintended interactions, if I'm reading all this correctly.